### PR TITLE
Init checkin

### PR DESCRIPTION
**Pull request by HarithaIBM**
Dependent library for GPG tool

### CODE REVIEW

File: buildenv [hunk 1]
```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.10.2.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libgpg"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_EXTRA_LDFLAGS="${ZOPEN_EXTRA_LDFLAGS} -L\$LIBGPG_HOME/lib"
```
Comment (IgorTodorovskiIBM) [inline buildenv:4]:
```suggestion
export ZOPEN_EXTRA_LDFLAGS="${ZOPEN_EXTRA_LDFLAGS} -L\$LIBGPGERROR_HOME/lib"
```
Should it be LIBGPGERROR_HOME with the name change or is it still LIBGPG_HOME?

Comment (HarithaIBM) [reply buildenv:4]: I have corrected this, thank you!

File: buildenv [hunk 2]
```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.10.2.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libgpgerror"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_EXTRA_LDFLAGS="${ZOPEN_EXTRA_LDFLAGS} -L\$LIBGPG_HOME/lib"
+export ZOPEN_EXTRA_LIBS="${ZOPEN_EXTRA_LIBS} -lgpg-error"
+export ZOPEN_EXTRA_CFLAGS="${ZOPEN_EXTRA_CFLAGS} -O0"
```
Comment (IgorTodorovskiIBM) [inline buildenv:6]: Does -O3 (the default opt) work?

File: buildenv [hunk 3]
```diff
@@ -0,0 +1,37 @@
+export ZOPEN_TARBALL_URL="https://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.10.2.tar.bz2"
+export ZOPEN_TARBALL_DEPS="make bzip2 tar curl perl automake autoconf m4 sed gettext zoslib diffutils libgpgerror"
+export ZOPEN_TYPE="TARBALL"
+export ZOPEN_EXTRA_LDFLAGS="${ZOPEN_EXTRA_LDFLAGS} -L\$LIBGPG_HOME/lib"
+export ZOPEN_EXTRA_LIBS="${ZOPEN_EXTRA_LIBS} -lgpg-error"
```
[the remark this answers is not in the document]
Comment (HarithaIBM) [reply buildenv]: Yes it is not required, i have removed it. Thank you, Igor!

File: buildenv (hunk 2 above)
Comment (HarithaIBM) [reply buildenv]: Yes , it works.

File: patches/g10lib.h.patch [hunk 4]
```diff
@@ -0,0 +1,13 @@
+diff --git a/src/g10lib.h b/src/g10lib.h
+index a7aee80..51a92f2 100644
+--- a/src/g10lib.h
++++ b/src/g10lib.h
+@@ -44,7 +44,7 @@
+ 
+ /* Attribute handling macros.  */
+ 
+-#if __GNUC__ > 2 || (__GNUC__ == 2 && __GNUC_MINOR__ >= 5 )
++#if ((__GNUC__ > 2 || (__GNUC__ == 2 && __GNUC_MINOR__ >= 5 )) && ! defined (__MVS__))
```
Comment (IgorTodorovskiIBM) [inline patches/g10lib.h.patch:10]: We can probably revisit this for clang

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**